packages/tally/src/token-addresses.ts

```diff
@@ -0,0 +1,10 @@
+import { type Address } from 'viem'
+import { ARB_ONE_CHAIN_ID } from './chain-ids.js'
+
+// For now we're mainly supported the highest traffic Arbitrum tokens
+// Support for USDC seems like it might be non-trivial
+// https://docs.arbitrum.io/bridge-tokens/concepts/usdc-concept
+// [DAI]
```

Comment:
[nit] I think this comment was carried over from Arbitrum bridge?

Reply:
100% good catch thanks